docs/about-core-concepts/architecture-guide.rst

```diff
@@ -10,7 +10,7 @@ USGS's Landsat Collection 2, Planet and a wide range of other organisations.
 The ODC can index from STAC, although this process is not fully integrated, it is
 done in production. An example of this is captured in these `Sentinel-2 Indexing notes`_. And
 in addition to indexing from STAC, `Datacube Explorer`_ can present ODC metadata as STAC
-documents through a STAC API.
+documents. through a STAC API.
 
 The key design constraint that the ODC currently has is it's reliance on a direct
 PostgreSQL connection, so one should consider how others in their team will access
```

Comment:
Is this language still suitable given that we'll be switching to PostgreSQL + PostGIS? Would it be more appropriate to perhaps explain that the ODC requires a connection to a database, specifically PostgreSQL? Perhaps this goes hand-in-hand with my comment above, and that the first paragraph/section of this page should introduce more generic architecture concepts (data, metadata, database) and their relationship to each other, and then go on to give more specific examples of those (COGs, STAC, PostgreSQL).